packages/lit-auth-client/src/lib/providers/BaseProvider.ts

```diff
@@ -175,27 +175,22 @@ export abstract class BaseProvider {
     }
 
     const litContracts = new LitContracts({
-      randomPrivatekey: true,
-      network: this.litNodeClient.config.litNetwork,
+      signer: new ethers.Wallet(ethers.Wallet.createRandom().privateKey, new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)),
+      network: LIT_NETWORK.DatilDev,
+      rpc: LIT_RPC.CHRONICLE_YELLOWSTONE
     });
-    try {
-      await litContracts.connect();
-    } catch (err) {
-      throw new UnknownError(
-        {
-          cause: err,
-        },
-        'Unable to connect to LitContracts'
-      );
-    }
 
+    console.log("🔄 Connecting to Lit Contracts...");
+    await litContracts.connect();
+    console.log("✅ Connected to Lit Contracts");
+
+    let pkps: IRelayPKP[] = [];
```

Comment:
```suggestion
    try {
      await litContracts.connect();
    } catch (err) {
      throw new UnknownError(
        {
          cause: err,
        },
        'Unable to connect to LitContracts'
      );
    }
```

We should use the original connection code, we don't need to change this

Reply:
It seems the `pkps` are now never returned